Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/FeedRecordedFuture.py

```diff
@@ -563,10 +574,11 @@ def main():  # pragma: no cover
     api_token = params.get('credentials_api_token', {}).get('password') or params.get('api_token')
     if not api_token:
         raise DemistoException('API Token must be provided.')
-    client = Client(RF_INDICATOR_TYPES[params.get('indicator_type')], api_token, params.get('services'),
-                    params.get('risk_rule'), params.get('fusion_file_path'), params.get('insecure'),
-                    params.get('polling_timeout'), params.get('proxy'), params.get('threshold'),
-                    params.get('risk_score_threshold'), argToList(params.get('feedTags')), params.get('tlp_color'))
+    client = Client(RF_INDICATOR_TYPES[params.get('indicator_type')], api_token, params.get('services'),  # type: ignore
```

Comment:
Why adding all this '# type:  ignore'?

Reply:
`# type: ignore` was needed while I was making changes from VS Code because of some errors it's debugger was throwing, they have now been removed from the final file